microprofile.jdt/org.eclipse.lsp4mp.jdt.core/META-INF/MANIFEST.MF

```diff
@@ -45,7 +45,8 @@ Export-Package: io.quarkus.runtime.util,
  org.eclipse.lsp4mp.jdt.internal.health.java;x-friends:="org.eclipse.lsp4mp.jdt.test",
  org.eclipse.lsp4mp.jdt.internal.metrics;x-friends:="org.eclipse.lsp4mp.jdt.test",
  org.eclipse.lsp4mp.jdt.internal.metrics.java;x-friends:="org.eclipse.lsp4mp.jdt.test",
- org.eclipse.lsp4mp.jdt.internal.restclient;x-friends:="org.eclipse.lsp4mp.jdt.test"
+ org.eclipse.lsp4mp.jdt.internal.restclient;x-friends:="org.eclipse.lsp4mp.jdt.test",
+ org.yaml.snakeyaml
```

Comment:
Should this have `x-friends:="org.eclipse.lsp4mp.jdt.test"`?

Reply:
You did that for test I suppose, if it that yes add it.